scipy/signal/_filter_design.py

```diff
@@ -1231,6 +1262,18 @@ def sos2tf(sos):
     Notes
     -----
     .. versionadded:: 0.16.0
+
+    Examples
+    --------
+    Find the polynomial representation of the transfer function H(s) 
```

Comment:
I think this needs rephrasing slightly as you are using an Elliptic filter not a given transfer function $H(s)$

Reply:
You are right! thx.